src/utils/utils.ts

```diff
@@ -0,0 +1,2 @@
+module.exports.makeOption = (name) => ({label: name, value: name});
+module.exports.makeOptions = (names) => names.map((name) => ({label: name, value: name}));
```

Comment:
Seems weird to use `export ...` in some cases, and use `module.exports...` here

```suggestion
export const makeOption = (name) => ({label: name, value: name});
export const makeOptions = (names) => names.map(makeOption);
```

Reply:
Correct.  I made have these changes in another PR but will make them here :)